adapters/eplanning/eplanning.go

```diff
@@ -196,14 +215,31 @@ func isMobileDevice(request *openrtb2.BidRequest) bool {
 	return request.Device != nil && (request.Device.DeviceType == adcom1.DeviceMobile || request.Device.DeviceType == adcom1.DevicePhone || request.Device.DeviceType == adcom1.DeviceTablet)
 }
 
+func getImpTypeRequest(request *openrtb2.BidRequest, totalImps int) int {
+
+	impType := impTypeBanner
+	for i := 0; i < totalImps; i++ {
+		imp := request.Imp[i]
+		if imp.Video != nil {
+			if imp.Video.Placement > 0 && imp.Video.Placement == vastInstream {
```

Comment:
Given that this is an `&&` statement, the second conditional seems to supersede the first. We can probably reduce to:
```
223         if imp.Video != nil {
224 -           if imp.Video.Placement > 0 && imp.Video.Placement == vastInstream {
    +           if imp.Video.Placement == vastInstream {
225                 impType = vastInstream
226             } else if impType == 0 {
227                 impType = vastOutstream
228             }
229         }
adapters/eplanning/eplanning.go
```